### PR TITLE
(QENG-2385) Use install_puppet to install puppet.

### PR DESCRIPTION
**Pull request by waynr**
As recommended by the Beaker developers.

### CODE REVIEW

**Comment by waynr** [comment]
Tested successfully using a `redhat6-64ma-debian6-64a-windows2008r2-64a` SUT layout.

**Comment by jeffmccune** [comment]
:+1:

File: acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb [hunk 1]
```diff
@@ -9,8 +11,10 @@
   if not puppet_version
     fail "PUPPET_LEGACY_VERSION is not set, e.g. '3.7.5'"
   end
-  install_package host, 'puppet', puppet_version
+  hosts = [host]
+  install_puppet({:version => puppet_version})
```
Comment (joshcooper) [inline acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb]: @anodelman @kevpl can we update beaker's `install_puppet` method to take an optional array of hosts, and not always assume you want to install puppet on all hosts? That way we wouldn't have to do terrible things like this.

Comment (waynr) [inline acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb]: @joshcooper I filed a ticket for this last night: https://tickets.puppetlabs.com/browse/BKR-263

**Comment by camlow325** [comment]
:+1: and merging - with the assumption that we'll come back around and fix this up after we have a new Beaker release with the work being done for BKR-263.  Thanks much, @waynr, for all of the work on this!

File: acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb (hunk 1 above)
Comment (joshcooper) [inline acceptance/suites/pre_suite/puppet3_compat/70_install_puppet.rb]: Just saw your comment, thanks @waynr!

**Pull request merged**